setup.py

```diff
@@ -61,6 +61,7 @@ def read(fname):
         'Natural Language :: English',
         'Operating System :: OS Independent',
         'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 3',
```

Comment:
Unsure whether we want to add this classifier at this stage. Although this repository itself might be Python 3 compatible, the library will remain largely non-functional on Python 3 until OMERO.py becomes Python 3 compatible.
Is there an official reference defining the meaning of this classifier so far.

Reply:
it is what you mean http://pypi.python.org/pypi?%3Aaction=list_classifiers